docs/conf/prague/2022/sponsors/prospectus.rst

```diff
@@ -0,0 +1,270 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/{{ shortcode }}-group.png
+
+Online Sponsorship Prospectus
```

Comment:
@ericholscher does this needs any adjustment?

Reply:
I don't think we changed things much from last year, so probably fine. Did we copy it from PDX or PRG last year?

Reply:
Yea, looks good to me. 👍